source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
+
+### Show and Tell
+
+- Near the beginning of each sprint we have a thirty-minute show and tell showcase that provides an opportunity to share our progress from the previous sprint with the wider organisation.
+- [TODO: show and tell content]
+
+#### Audience
+
+All of Data Platform and any wider audience
+
+### Roadmap Review
+
+- ... share with our colleagues and the wider organisation ...
+- ... consists of an ... what’s next in the road map
+
+#### Audience
+
+All of the Data Improvement Programme and any wider audience
+
+### Refinement
+
+**Purpose:**
+
+We use refinement to align the team's understanding and agree on the units of work for upcoming sprints.
+It's where we ensure everyone has an understanding of the backlog items before they are added to a sprint.
+
+**Preparation:**
+
+Here's what can be prepared prior to engaging in a refinement session:
+
+- **Backlog overview:**
+Review the backlog to get a sense of what's coming up, and what's already been done.
+Read the tickets likely to come up for discussion, so you know what questions you want to ask.
+- **Backlog item review:**
+Ensure any tickets are well fleshed out and ready for discussion.
+This should include a detailed description of the story, as well as acceptance criteria.
+- **Understand Dependencies:**
+If your tickets rely on other work already being done,
+know which tickets are required and list them in the body of the issue.
+This helps the team establish sequencing of the tickets during refinement.
+- **Prepare to Communicate!:**
+Be prepared to articulate why you think a ticket should be done, how big you think it is,
+be receptive to teammates, and collaborate to achieve a shared understanding of the tasks at hand.
+Be open to suggestions on how your ticket could be improved for someone picking it up.
+
+**Process:**
+
+- **Selection:**
+Before refinement, the PM picks out the stories for the team to discuss.
+- **Discussion:**
+Dive into the details of each story, hash out the acceptance criteria,
+and call out any dependencies or risks.
+- **Estimation:**
+Estimate based on effort and risk using a [planning-poker scoring system](https://en.wikipedia.org/wiki/Planning_poker#Process), allowing the team to have a clear sense of the size and scope of each ticket.
+- **Decompose:**
+Chop up any stories that are too big to handle in a sprint into smaller bits.
+- **Define 'Done':**
+Ensure each story has a clear _Definition of Done_ (DoD) to know exactly when it is crossed the finish line.
+
+**Output:**
+
+- **Refined backlog:**
+Stories that are fleshed out, estimated, and ready for the PM to prioritize
+- **Scored stories:**
+Each story gets a score so we can try to estimate how many stories we should take on based on our prior rate of work.
+- **Approachable tickets:**
+All tickets that go through refinement should have all the info required for any member of the team to pick up.
+
+#### Audience
+
+Data Catalogue Team
+
+### Peer Review
```

Comment:
Are we still doing this?

Reply:
Ah, I've put down the Data Platform's peer review, but I thought it was a Data Catalogue peer review. Not sure if that actually goes ahead though